classes/Carrier.php

```diff
@@ -72,7 +72,7 @@ class CarrierCore extends ObjectModel
     /** @var bool True if extra shipping handling cost should be applied to this Carrier */
     public $shipping_handling = true;
 
-    /** @var bool Behavior taken for unknown range */
+    /** @var bool Behavior for out-of-range weights : true to disable carrier, false to apply the cost of the highest defined range */
```

Comment:
```suggestion
    /** @var bool Behavior for out-of-range weights: true to disable carrier, false to apply the cost of the highest defined range */
```

remember about no space before `:` :D

Reply:
🆙 @thomasnares

Reply:
Oups thanks, didn't got the notif